### ISSUE TITLE
l cant down load

### ISSUE THREAD

**Issue by crazyoldhippieguy** [catseye/Chrysoberyl#5]
help, l cant down load your alpaca program, please e-mail me it, billbusuttil@t-online.de Thanck you.

**Comment by Zomis** [comment]
I think what you can do is to make the link http://catseye.tc/distfiles/ more accessible.

I started out on http://catseye.tc/node/pibfi , then I got stuck on http://catseye.tc/node/interpreter and other pages on /node, then I found http://catseye.tc/node/Distribution and at the bottom of the page it said "generated by..." and that's how I found this Github repository, and then I found this issue. This issue contains the only link to http://catseye.tc/distfiles/ as far as I have seen. And even finding this Github repo wasn't easy. Not even http://catseye.tc/node/Chrysoberyl has a clear link to GitHub.

**Comment by cpressey** [comment]
@Zomis Thanks for the feedback.  I grant that the navigation on the site isn't fantastic.  But most entries for implemented languages (and other implemented things) do have links to distfiles on their page.  For example, on http://catseye.tc/node/ALPACA , in the right-hand column half-way down the page, there's a heading "Download" and a link "Download latest release" with a blue outline right below it.  And a link to GitHub below that, and links to previous versions below that.

pibfi turns out to be an exception because tbh, there are arguably too many implementation of brainfuck out in the world already, and I just don't care about it.  The fact that it's even got an entry is more out of completeness than anything.

Also, the distribution structure was deprecated at one point, and I had no idea the Distribution node even existed anymore.  I agree, it is practically useless now that those things have nothing to link to...

I'll see what I can do about making the association between each thing and where it lives on GitHub more obvious, as time permits.  Thanks again!

**Comment by cpressey** [comment]
The structure of Chrysoberyl, and the http://catseye.tc/ website, have been completely overhauled since this issue was created and the above comments were written.

The new approach lists many entries on a single-page article, addressed by HTML anchors, for example: http://catseye.tc/article/Languages.md#alpaca

Each of those entries that has a repository on GitHub, has an "on GitHub" button on the right which links to it, as well as a "Download" button which links to a page that lists all of the distributions' zip files available for download (in ALPACA's case, [this page](http://catseye.tc/distribution/ALPACA_distribution)).

Entries that do not have a repository on GitHub may still have an "in distribution" property which links to their distribution page.

In summary, whether the new structure and layout is better or worse, I think it provides the things requested by this issue (clear links to GitHub and to download distfiles for projects), so I'm closing this issue.  Thank you for your feedback.
